submitted_models/marble_husky_sensor_config_3/specifications.md

```diff
@@ -0,0 +1,86 @@
+The MARBLE Husky has the following estimate commercial cost components:
+* Base vehicle: $25,000
+* Sensor suite: $12,000
+* Compute/Support Electronics: $4,000
+* Total: ~ $41,000
+
+Its weight is approximately 85 lbs (two-person heft).
+
+### Sensors
+This MARBLE Husky with sensor configuration 1 includes the following sensors. The specifications for these instruments are provided below in the [Validation Links](#validation_links) section.
+
+The following specific sensors are declared payloads of this vehicle:
+
+* Trossen ScorpionX MX-64 Robot Turret, modeled by JointStateController and JointStatePublisher plugins.
+* D435i RGBD Camera (x3), modeled by rgbd_camera plugin
+  - 1x fixed, forward-looking
+  - 1x fixed, downward-facing at about 45 degrees (for examining terrain)
+  - 1x gimballed (on the Trossen turret)
+* Ouster 3D Lidar (64 Channel), modeled by gpu_lidar plugin
+* Microstrain IMU: 3DM-GX5-25, modeled by imu_sensor plugin. Notes on modeling of the IMU are included in the model.sdf file.  (located under the 3D lidar, installed at same x,y location as 3D lidar)
+- RPLidar S1 Planar Lidar (under the 3D lidar), modeled by gpu_ray plugin
+- Vividia HTI-301 LWIR Camera (not modeled because thermal camera not yet supported in simulator) - located on the turret next to the D435i and light.  
+
+### Control
+This MARBLE Husky is controlled by the DiffDrive plugin.  It accepts twist inputs which drive the vehicle along the x-direction and around the z-axis.  
+
+### Motion CharacteristicsBased on the tests specified in the DARPA SubT Challenge [Model PreparationGuide](https://subtchallenge.com/\<fix_me\>), this vehicle has the following motion constraint characteristics.
+
+This configuration has the same motion characteristics as the COSTAR husky, except it also has a pan/tilt mechanism which has additional motion characteristics.  We have included a test script in this folder (test_gimbal.sh) which sends pan and tilt commands using ROS topics.  This script assumes you have named your vehicle X1.  
+
+The motion characteristics are the same as the COSTAR husky which has already been modeled.  We were unable to perform additional validation tests due to COVID-19 restrictions.  
+
+### Endurance Characteristics
+This configuration has an endurance of approximately 4 hours, but we have limited the model in simulation to have a 2 hour endurance.  We plan to carry out the endurance test characterization but have been prevented from doing so due to the coronavirus measures preventing us from visiting the lab space while preparing these models for simulation.  
+
+### Diversions from Physical Hardware of MARBLE Husky
+Computers were installed in the payload area of the husky and these have been roughly modeled (the rail and computer bay are shown in the model.sdf model).  The MARBLE vehicle uses an AMD Ryzen processor (32-core) with 64 GB of RAM.  It has a cooling system as well.  
+
+## X4 Validation and Specification Links
```

Comment:
X4 should be marble?

Reply:
This has been changed to marble husky. Thanks for the catch!